sklearn/manifold/t_sne.py

```diff
@@ -142,10 +142,12 @@ def _kl_divergence(params, P, degrees_of_freedom, n_samples, n_components,
 
     # Q is a heavy-tailed distribution: Student's t-distribution
     n = pdist(X_embedded, "sqeuclidean")
-    n += 1.
```

Comment:
Could you show us your calculation? I think the formulas come from the [parametric t-SNE paper](https://lvdmaaten.github.io/publications/papers/AISTATS_2009.pdf). How does it compare to that?